models/tokenlon/ethereum/tokenlon_v5_ethereum_rfq_v2_trades.sql

```diff
@@ -0,0 +1,109 @@
+{{ config(
+    schema = 'tokenlon_v5_ethereum',
+    alias = 'rfq_v2_trades',
+    
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address'],
+    post_hook = '{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "tokenlon",
+                                    \'["keen"]\') }}'
+    )
+}} 
+{% set blockchain = 'ethereum' %}
+{% set project = 'tokenlon' %}
+{% set project_version = '5' %}
+-- block
+{% set project_start_date = "timestamp '2023-09-12'" %}
+
+WITH dexs AS (
+    SELECT 
+        evt_block_time        AS block_time,
+        "user"                AS taker,
+        "maker"               AS maker,
+        "takerTokenAmount"    AS token_sold_amount_raw,
+        "makerTokenAmount"    AS token_bought_amount_raw,
+        cast(NULL as double)  AS amount_usd,
+        "takerToken"          AS token_sold_address,
+        "makerToken"          AS token_bought_address,
+        contract_address      AS project_contract_address,
+        evt_tx_hash           AS tx_hash,
+        CAST(ARRAY[] as array<bigint>) AS trace_address,
```

Comment:
We may use CAST(ARRAY[-1] as array<bigint>) for null trace_address here:
```suggestion
        CAST(ARRAY[-1] as array<bigint>) as trace_address,
```